tests/e2e.rs

```diff
@@ -302,6 +310,35 @@ async fn test_e2e_ldaps_starttls() {
 	assert!(user.is_some());
 }
 
+#[test(tokio::test)]
+#[test_log(default_log_filter = "debug")]
+async fn test_e2e_no_phone() {
+	let mut ldap = Ldap::new().await;
+	ldap.create_user("Bob", "Tables", "Bobby", "no_phone@famedly.de", None, "no_phone", false)
+		.await;
+
+	sync_ldap_users_to_zitadel(config().await.clone()).await.expect("syncing failed");
+
+	let zitadel = open_zitadel_connection().await;
+	let user = zitadel
+		.get_user_by_login_name("no_phone@famedly.de")
+		.await
+		.expect("could not query Zitadel users");
+
+	assert!(user.is_some());
+
+	if let Some(user) = user {
```

Comment:
Since `user` is asserted to be `is_some`, we can just unwrap here. Or `expect`: `let user = user.expect("no user")`